src/dynamics/ads_player/ads_player.js

```diff
@@ -386,11 +386,27 @@ Scoped.define("module:Ads.Dynamics.Player", [
                     }
                     return true;
                 },
+                getMediaUrl: function(adObj) {
+                    let adMediaUrl = adObj?.data?.mediaUrl;
```

Comment:
for the example vast in comments, the mediaUrl is null so im attempting to get the mediaUrl another method as a backup plan